tempesta_fw/http_sticky.c

```diff
@@ -232,8 +240,10 @@ tfw_http_sticky_set(TfwHttpMsg *hm)
 	+ STICKY_NAME_MAXLEN + 1 + STICKY_KEY_MAXLEN * 2 + 2
 
 static int
-tfw_http_sticky_add(TfwHttpMsg *hm, u_char *value, size_t len)
+tfw_http_sticky_add(TfwHttpMsg *hmresp, TfwHttpMsg *hmreq)
 {
+	TfwClient *client;
+	unsigned int len = sizeof(client->cookie.hmac);
 	char buf[len * 2];
 	TfwStr set_cookie = {
```

Comment:
Bad stack memory alignment: 8, 4, 40 and 24 bytes are allocated. It's better to place them like 4, 8, 24, 40 to minimized 8-byte alignment